airbyte-ci/connectors_ci/metadata_service/lib/tests/fixtures/invalid/metadata_catalog_no_id_override.yaml

```diff
@@ -18,4 +18,3 @@ data:
     oss:
       enabled: true
       definitionId: woohoo
-
```

Comment:
Looks like all the files in this folder are showing with whitespace changes. Are they really needed?

Reply:
Same as above. I only made changes in 1 file, and the rest were automatically committed. Commit: https://github.com/airbytehq/airbyte/pull/24207/commits/b2e4570f3f411b787e9e98b590377c79b36aad32